### ISSUE TITLE
Drush does not see Drupal installation

### ISSUE THREAD

**Issue by Chi-teck** [drush-ops/drush#2861]
It worked correctly in Drush 9 beta1

```
$ /tmp/drush-9.0.0-beta1/vendor/bin/drush st
 Drupal version   : 8.4.0-dev                                    
 Site URI         : http://www                                   
 DB driver        : mysql                                        
 DB hostname      : localhost                                    
 DB port          : 3306                                         
 DB username      : root                                         
 DB name          : d8_nimfa                                     
 Database         : Connected                                    
 Drupal bootstrap : Successful                                   
 Default theme    : bartik                                       
 Admin theme      : seven                                        
 PHP binary       : /usr/bin/php7.1                              
 PHP config       : /etc/php/7.1/cli/php.ini                     
 PHP OS           : Linux                                        
 Drush script     : /var/www/d8/docroot/vendor/drush/drush/drush 
 Drush version    : 9.0-dev-g9a77e4d                             
 Drush temp       : /tmp                                         
 Drush configs    :                                              
 Drush aliases    :                                              
 Install profile  : standard                                     
 Drupal root      : /var/www/d8/docroot/                         
 Site path        : sites/default                                
 Files, Public    : sites/default/files                          
 Files, Temp      : /tmp                                         

$ /tmp/drush-9.0.0-beta2/vendor/bin/drush st
 PHP binary    : /usr/bin/php7.1                         
 PHP config    : /etc/php/7.1/cli/php.ini                
 PHP OS        : Linux                                   
 Drush script  : /tmp/drush-9.0.0-beta2/vendor/bin/drush 
 Drush version : 9.0.0-beta2                             
 Drush temp    : /tmp                                    
 Drush configs :                                         
 Drush aliases :         
```

**Comment by weitzman** [comment]
In the first request, how is Drush finding  /var/www/d8/docroot/? Whats your cwd?

**Comment by Chi-teck** [comment]
CWD is Drupal root directory (/var/www/d8/docroot/).

**Comment by webflo** [comment]
Released drupal-finder 0.4.2 a few minutes ago. This should fix the regression.
